libp2p/pubsub/pubsub.py

```diff
@@ -336,6 +403,14 @@ def handle_subscription(self, origin_id: ID, sub_message: Any) -> None:
             return
 
         # TODO: - Validate the message. If failed, reject it.
+        # Validate the signature of the message
+        # FIXME: `signature_validator` is currently a stub.
+        if not signature_validator(msg.key, msg.SerializeToString()):
+            return
```

Comment:
This early exit seems like a prime place for a logging message.

Reply:
There was #119 that implements logging. Besides the standard logging, he also implement some of his ideas there. Wonder if you have time to take a look. @pipermerriam

Reply:
I'd say I lean :-1: on the fancy stuff in #119 as it tries to automate something that I don't think automates well.  Good logging messages are "human" things, at least in this context where we're trying to understand the sequence of events with minimal noise.

Similarly there is performance overhead to logging so an approach that does something like drop a decorator on every function and log all the stuff it does is likely to have a measurable effect on speed.

I'd be much more supportive of just giving the various classes their own loggers and putting some logging messages in the places that feel important/significant and then we can expand from there.

I don't think there's a direct need for `DEBUG2` level logs or anything like that at this stage and it's very easy to add in later once the need is clear.